### PR TITLE
doc: Update year in footer

### PR DESCRIPTION
**Pull request by CharlesKasasira**
## What kind of change does this PR introduce?

docs update

## What is the current behavior?

issue: #13859 
The year is still 2022 in the footer

## What is the new behavior?

The year is generated dynamically.

### CODE REVIEW

**Review by gregnr — APPROVED** [review]
Thanks for catching this @CharlesKasasira 😃

File: apps/docs/components/Navigation/Footer.tsx [hunk 1]
```diff
@@ -11,7 +11,7 @@ const Footer = () => {
       <hr className="border-scale-400  mt-8"></hr>
       <div className="flex gap-4 items-center mt-6 justify-between">
         <div className="flex flex-col lg:flex-row gap-3 ">
-          <span className="text-xs text-scale-900">Supabase 2022</span>
+          <span className="text-xs text-scale-900">Supabase {new Date().getFullYear()}</span>
```
Comment (MildTomato) [inline apps/docs/components/Navigation/Footer.tsx:14]: not actually sure this is meant to be here at all.
usually this is a copyright reference.

Comment (CharlesKasasira) [reply apps/docs/components/Navigation/Footer.tsx]: I have updated the PR to remove the year and link to the marketing page as suggested.

File: apps/docs/components/Navigation/Footer.tsx [hunk 2]
```diff
@@ -11,7 +11,9 @@ const Footer = () => {
       <hr className="border-scale-400  mt-8"></hr>
       <div className="flex gap-4 items-center mt-6 justify-between">
         <div className="flex flex-col lg:flex-row gap-3 ">
-          <span className="text-xs text-scale-900">Supabase 2022</span>
+          <Link href="https://supabase.com/">
+            <a className="text-xs text-scale-900">Supabase</a>
+          </Link>
```
Comment (Isaiah-Hamilton) [inline apps/docs/components/Navigation/Footer.tsx:16]: I don't think this is what @MildTomato meant. I think he meant for example on the marketing site we do `© Supabase Inc` in the footer. Attached an image below of the marketing site footer for reference.

CleanShot 2023-04-26 at 16 38 43@2x

Comment (CharlesKasasira) [reply apps/docs/components/Navigation/Footer.tsx]: Okay, I did not think of that, I have just updated it.

Screenshot 2023-04-27 at 00-34-41 Supabase Docs

**Review by Isaiah-Hamilton — APPROVED** [review]

**Pull request merged**